portfolio.adobe.com.prod.json

```diff
@@ -0,0 +1,40 @@
+{
+  "providerId":"portfolio.adobe.com",
+  "providerName":"Adobe Portfolio",
+  "serviceId":"prod",
+  "serviceName":"Adobe Portfolio Websites",
+  "version":1,
+  "description":"Enables a domain to work with Adobe Portfolio",
```

Comment:
Maybe `Configures a domain to point to Adobe Portfolio`?

Reply:
Really, I am just going with the convention that other service providers have used:
https://github.com/Domain-Connect/Templates/blob/master/shopify.com.website.json
https://github.com/Domain-Connect/Templates/blob/master/wix.com.wixsites.json

(Also, this isn't _technically_ the configuring portion of the process, just the data that allows DNS hosts to try to configure a domain.)